### PR TITLE
Update model url from aws to aliyun

### PR DESCRIPTION
**Pull request by hhaAndroid**
update model url from aws to aliyun

### CODE REVIEW

File: demo/inference_demo.ipynb [hunk 1]
```diff
@@ -82,7 +82,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.7"
+   "version": "3.8.0"
```
Comment (ZwwWayne) [inline demo/inference_demo.ipynb:85]: This modification is unnecessary

File: mmdet/models/roi_heads/test_mixins.py [hunk 2]
```diff
@@ -148,6 +148,9 @@ async def async_test_mask(self,
             if det_bboxes.shape[0] == 0:
                 segm_result = [[] for _ in range(self.mask_head.num_classes)]
             else:
+                if rescale and not isinstance(scale_factor,
+                                              (float, torch.Tensor)):
+                    scale_factor = det_bboxes[:, :4].new_tensor(scale_factor)
```
Comment (ZwwWayne) [inline mmdet/models/roi_heads/test_mixins.py:153]: `[:, :4]` is unnecessary

File: demo/inference_demo.ipynb [hunk 3]
```diff
@@ -18,7 +18,7 @@
    "source": [
     "config_file = '../configs/faster_rcnn/faster_rcnn_r50_fpn_1x_coco.py'\n",
     "# download the checkpoint from model zoo and put it in `checkpoints/`\n",
```
Comment (ZwwWayne) [inline demo/inference_demo.ipynb:20]: Tell users the exact download link in the comments.

**Comment by hellock** [comment]
Better to split this PR into two. One PR for one thing.

File: tests/async_benchmark.py [hunk 4]
```diff
@@ -80,17 +82,15 @@ async def detect(img):
             ]
 
     result_dir = os.path.join(project_dir, 'demo')
-    show_result(
+    model.show_result(
```
Comment (hellock) [inline tests/async_benchmark.py:85]: This modification is irrelevant to model urls. It can be fixed in another PR.

**Review by hellock — APPROVED** [review]

**Pull request merged**